ports/stm32/machine_i2s.c

```diff
@@ -125,6 +127,13 @@ typedef struct _non_blocking_descriptor_t {
     bool copy_in_progress;
 } non_blocking_descriptor_t;
 
+typedef struct _i2spll_config_t {
```

Comment:
I suggest calling this struct `_pll_config_i2s_t`.

Reply:
The I2S PLL is referred to as `plli2s` throughout the stm32 reference docs. This led to the choice of `_plli2s_config_t`.